gistUploader.js

```diff
@@ -1,11 +1,20 @@
-
 const axios = require('axios');
+const fs = require('fs');
+
+const API_URL = 'https://api.github.com/gists';
+const fileName = process.argv[3];
```

Comment:
This variable name can lead to some confusion, because  argv[2] is another file name, right?

Reply:
done